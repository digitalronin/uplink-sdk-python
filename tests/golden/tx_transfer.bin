00000000: 03 EC 2D 37 7F D1 0E 64  34 AF 54 AC E3 A3 8E 14  ..-7...d4.T.....
00000010: 52 C3 1A 09 43 0E 82 50  58 09 A9 CA 5D 69 EF 24  R...C..PX...]i.$
00000020: 94 C6 64 86 E7 77 43 09  64 01 FD AF C6 D7 60 D8  ..d..wC.d.....`.
00000030: 90 4E 01 39 79 5F 74 00  84 A8 51 2F 92 75 48 66  .N.9y_t...Q/.uHf
00000040: 82 1D 00 00 00 00 00 00  00 05                    ..........