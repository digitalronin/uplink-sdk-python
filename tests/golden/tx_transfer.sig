r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 18206379320156371196429836960309880825273030767310817840103301167733763710747
