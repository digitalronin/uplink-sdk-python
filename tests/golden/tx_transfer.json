{
    "timestamp": 1231006505, 
    "to": null, 
    "header": {
        "tag": "TxAsset", 
        "contents": {
            "tag": "CreateAsset", 
            "contents": {
                "balance": 5, 
                "assetAddr": "43WRxMNcnYgZFcE36iohqrXKQdajUdAxeSn9mzE1ZedB", 
                "toAddr": "7mR5d7s6cKB4qjuX1kiwwNtygfURhFQ9TKvEd9kmq6QL"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE05NDIwNDYxODgxNzY0NzQzNjY3MDUyNTg3NDQ4NTU4NDUxMzY1NzE0MjMzNzAxODY0MjQxNDE4MjA2NDk3MDg5MzU1OTUwNDEyMjg4Mg==", 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
}